bin/main_pysam.py

```diff
@@ -157,11 +160,12 @@ def main():
             out_raw["other_allele"] = ss_rec.hm_other_al.str() if vcf_rec and ss_rec.is_harmonised else args.na_rep_out
             out_raw["beta"] = ss_rec.beta if ss_rec.beta is not None and ss_rec.is_harmonised else args.na_rep_out
             out_raw["odds_ratio"] = ss_rec.oddsr if ss_rec.oddsr is not None and ss_rec.is_harmonised else args.na_rep_out
+            out_raw["z_score"] = ss_rec.zscore if ss_rec.zscore is not None and ss_rec.is_harmonised else args.na_rep_out
             out_raw["ci_lower"] = ss_rec.oddsr_lower if ss_rec.oddsr_lower is not None and ss_rec.is_harmonised else args.na_rep_out
             out_raw["ci_upper"] = ss_rec.oddsr_upper if ss_rec.oddsr_upper is not None and ss_rec.is_harmonised else args.na_rep_out
             out_raw["effect_allele_frequency"] = ss_rec.eaf if ss_rec.eaf is not None and ss_rec.is_harmonised else args.na_rep_out
             # Process the neg_log_10_p_value
-            if "neg_log_10_p_value" in out_header:
+            if tag_neg_log_10_p_value == True:
```

Comment:
`if tag_neg_log_10_p_value:` should be enough.

Reply:
Yes. The statement "if tag_neg_log_10_p_value:" is sufficient. Just in case this variable is misused or assigned any other value, I set it to `==True`.